workshop/content/60-go/20-create-project/500-deploy.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "cdk deploy"
+weight = 500
++++
+
+Okay, we've got a CloudFormation template. What's next? __Let's deploy it into our account!__
+
+## Bootstrapping an environment
+
+The first time you deploy an AWS CDK app into an environment (account/region),
+you can install a "bootstrap stack". This stack includes resources that
+are used in the toolkit's operation. For example, the stack includes an S3
+bucket that is used to store templates and assets during the deployment process.
+
+You can use the `cdk bootstrap` command to install the bootstrap stack into an
+environment:
+
+```
+cdk bootstrap
+```
+
+Then:
+
+```
+ ⏳  Bootstrapping environment aws://999999999999/us-east-1...
+...
+```
+
+{{% notice info %}} You might see an Access Denied error at this step,
```

Comment:
suggest change in wording

---

If you are returned an Access Denied message at this step, verify that

* You have configured the AWS CLI correctly (or, specified an appropriate secret/access key),
* You have permission to call `cloudformation:CreateChangeSet` within the scope of your account/session.

---

Reply:
Can't do the bullet point formatting in the notice as far as I'm aware, but I've updated the wording